ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -122,6 +122,19 @@ tests:
     test:
     - ref: openshift-extended-test
     - ref: rosa-terraform-e2e-tests-upload-polarion
+- as: rosa-sts-ad-critical-high-f3
+  cron: 38 11 3,6,9,12,15,18,21,23,27,30 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CASE_LABEL_FILTER: (Critical,High)&&(day1-post,day2)&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-sts-ad
```

Comment:
If possible, in the next step, I advise to use `TEST_PROFILE` instead of `CLUSTER_PROFILE` to avoid confusion with the prow ci `cluster_profile`.